docs/next.config.js

```diff
@@ -0,0 +1,9 @@
+/** @type {import('next').NextConfig} */
+module.exports = {
+  reactStrictMode: false,
+  typescript: { ignoreBuildErrors: true },
+  experimental: {
+    appDir: true,
```

Comment:
Do we still need this now that the app router is stable? I don't actually know 🤷